frontend/src/core/comments/components/CommentsList.tsx

```diff
@@ -66,14 +69,17 @@ export default function CommentsList(props: Props): React.ReactElement<'div'> {
                             renderComment(comment),
                         )}
                     </ul>
-
-                    <Localized id='comments-CommentsList--all-comments'>
-                        <h2 className='title'>ALL COMMENTS</h2>
-                    </Localized>
+                    {!produceAllComments ? (
```

Comment:
Seems like we "produce" All Comments if `produceAllComments` is **False**? We should either rename the flag to reflect that (hide instead of produce?) or invert the logic of `produceAllComments`.

Reply:
got it!